internal/entrypoint/entrypoint.go

```diff
@@ -0,0 +1,26 @@
+package entrypoint
+
+var ImageRef = "ghcr.io/chainguard-dev/terraform-provider-imagetest:latest"
```

Comment:
this is wrong, I'll update this once we have a release of entrypoint published, and move things to use an ldflag for setting this in the provider builds